packages/fast-component-explorer/app/msft-component-helpers/design-system.schema.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Design System",
+    "description": "A design system's schema definition.",
+    "id": "design-system",
+    "type": "object",
+    "properties": {
+        "designSystem": {
```

Comment:
Sorry if I missed it - but should we have a test that validates the default design system from `fast-components-styles-msft` against this schema, so that it will fail if changes are made to the design system but not to this schema?

Reply:
We do not yet have that, currently no testing is present for any component interface against a schema, we should do this all-up as a single set of tasks